### ISSUE TITLE
Broken

### ISSUE THREAD

**Issue by kmudrick** [kmudrick/morphir-uuidv5#1]
Hey @DamianReeves am I correct that [this](https://github.com/kmudrick/morphir-uuidv5/blob/main/src/Morphir/Uuidv5/App/Example.elm) would not be morphir-compatible because of these imports?

```elm
import UUID exposing (UUID, Error)
import Bytes exposing (Bytes)
import Bytes.Encode
```

Running `elm make src/Morphir/Uuidv5/App/Example.elm` works but `morphir-elm make` gives Error:

```
Error: [
  [
    "ResolveError",
    [
      [
        "example"
      ]
    ],
    [
      "NoMorphirPackageFoundForElmModule",
      [
        "Bytes",
        "Encode"
      ]
    ]
  ]
]
```

Are only the morphir sdk modules and `elm/core` importable in general in all morphir elm declarations?

**Comment by DamianReeves** [comment]
Yeah since Bytes is not one of the modules Morphir currently supports; however that can be changed and added as a feature.
